### ISSUE TITLE
Can't query many servers

### ISSUE THREAD

**Issue by user23145** [FunnyItsElmo/PHP-Minecraft-Server-Status-Query#27]
When I query some of the biggest servers out there, they all show up as "offline", when they're not.

Here are some of the servers I'm talking about:

* mineverse.com
* play.gotpvp.com
* mc.desiredcraft.net
* pvp.originmc.org

Anyone else have issues querying these servers? Why?

**Comment by Doc94** [comment]
Have you tried a localhost?
Because I use domain / dns on at least a 200 sv online and runs smoothly ...

Unless you have a query off ...

**Comment by ggtools** [comment]
I made a couple of tests since I have the issue with my private servers. The four servers above are indeed not working from the latest version from master.

I add some messages in the library and found the following facts:

- for my servers and the first and third server the initial packet is too short (0 for mine, 1 for the servers above)
- the second and fourth servers I got the following error on the message of the day:

    Catchable fatal error: Object of class stdClass could not be converted to string in /usr/src/myapp/MinecraftServerStatus/status.class.php on line 77

Since I add some debug lines line 77 on my script is:

    $motd = preg_replace("/(§.)/", "",$motd);

I looked at the message of the day for `play.gotpvp.com` and I'm not really surprised at all since the complex message of the day might be *hackish* and not translated as a string in PHP. For the other cases I have absolutely no idea since my servers are stock servers without and specific customization.
